docs/how-to-guides/others/an-example-procedure-for-adding-a-new-node.md

```diff
@@ -0,0 +1,89 @@
+# An example procedure for adding a new node - Case of localization
+
+## Overview
+
+This page provides a guide for evaluating Autoware when a new node is implemented, especially about developing a novel localization node.
+
+To keep things simple, an example of operation within AWSIM, as opposed to actual hardware, is provided.
+The same procedure could likely apply when operating on real hardware.
```

Comment:
I think this page does not strongly assume that users will use AWSIM. How about this suggestion?  
```suggestion
The workflow involves initial testing and rosbag recording using a real vehicle or AWSIM, implementing the new node, subsequent testing using the recorded rosbag, and finally evaluating with a real vehicle or AWSIM.
```